docs/guide/troubleshooting.md

```diff
@@ -12,26 +12,17 @@ You followed the [Getting started guide](/guide/getting-started.md) but you stil
 
 These are the most common reasons why you might not be able to see your scene:
 
-### Make sure you have a camera 🎥
+~~### Make sure you have a camera 🎥~~
```

Comment:
Since this is not an issue anymore, I'd suggest removing this paragraph from the troubleshooting docs entirely. One could argue, that this is useful for users prior to 2.2, but if they came across this they added a camera anyways 😄.

Reply:
It was more for the "Oh they are fixing stuff" effect